src/qml/imports/Theme/qmldir

```diff
@@ -1,6 +1,7 @@
 module Theme
 singleton Theme 1.0 Theme.qml
 QfCloseButton 1.0 QfCloseButton.qml
+QfToggleButtons 1.0 QfToggleButtons.qml
```

Comment:
@mohsenD98 , this should be taken out right?

Reply:
yes.